README.md

```diff
@@ -171,18 +171,48 @@ This command will start three instances of the adaptor behind a load balancer on
 To change the scale number while all services are running run the same "up" command with new scale value and then
 restart the load balancer container (so it will become aware of instance count change).
 
+### Running static checks
+
+**All static checks**
+    
+    ./gradlew check -x test -x integrationTest
```

Comment:
Not a task for this documentation PR, but what do you think about a new gradle task such as `quality` or more literally `staticCheck` as an alias for this?

Reply:
I agree with you and I don't think it would take a lot of effort to do. I like the name `quality` for the gradle task as I feel like `staticCheck` only really covers the Spotbugs tasks

Reply:
That's a good point, in which case "All static checks" may need to be revisited as a header.

Reply:
What do we think of the header name 'Quality Checks'?

I feel like it summarises what we're trying to achieve by using checkstyle & spotbugs

Reply:
Done in e315bcd